test/JDBC/expected/test_db_collation-vu-verify.out

```diff
@@ -0,0 +1,1800 @@
+GO
+~~START~~
+varchar#!#varchar
+ct#!#bbf_unicode_cp1_ci_as
+~~END~~
+
+
+-- Case 4.1: Computed columns
+USE test_db_collation_vu_prepare_db121;
+GO
+
+SELECT * FROM test_db_collation_vu_prepare_db121_computed_columns ORDER BY a;
+GO
+~~START~~
+nvarchar#!#nvarchar
+Año Nuevo#!#oveuN oñA
+café#!#éfac
+chaptéR#!#Rétpahc
+película#!#alucílep
+~~END~~
+
+
+SELECT * FROM test_db_collation_vu_prepare_db121_computed_columns ORDER BY b;
+GO
+~~START~~
+nvarchar#!#nvarchar
+película#!#alucílep
+café#!#éfac
+Año Nuevo#!#oveuN oñA
+chaptéR#!#Rétpahc
+~~END~~
+
+
+-- Case 4.2: Temp Tables
+CREATE TABLE #test_db_collation_vu_prepare_db121_temp_t1(a nvarchar(11), b nvarchar(11), c AS sys.reverse(b) NOT NULL)
+GO
+
+INSERT INTO #test_db_collation_vu_prepare_db121_temp_t1 VALUES
+('café', 'jalapeño'),
+('résumé', 'naïve'),
+('Piñata', 'Año Nuevo'),
+('TELÉFONO', 'película'),
+('árbol', 'canapé'),
+('chaptéR', 'TEññiȘ');
+GO
+~~ROW COUNT: 6~~
+
+
+SELECT * FROM #test_db_collation_vu_prepare_db121_temp_t1 WHERE c = 'evian'; -- reverse of naive
+GO
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar
+résumé#!#naïve#!#evïan
+~~END~~
+
+
+SELECT * FROM #test_db_collation_vu_prepare_db121_temp_t1 ORDER BY c;
+GO
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar
+TELÉFONO#!#película#!#alucílep
+árbol#!#canapé#!#épanac
+résumé#!#naïve#!#evïan
+café#!#jalapeño#!#oñepalaj
+Piñata#!#Año Nuevo#!#oveuN oñA
+chaptéR#!#TEññiȘ#!#ȘiññET
+~~END~~
+
+
+CREATE TABLE #test_db_collation_vu_prepare_db121_temp_t2(a nvarchar(11), b nvarchar(11), c AS sys.reverse(b) NOT NULL)
+GO
+
+INSERT INTO #test_db_collation_vu_prepare_db121_temp_t2 VALUES
+('café', 'jalapeño'),
+('résumé', 'naïve'),
+('Piñata', 'Año Nuevo'),
+('TELÉFONO', 'película'),
+('árbol', 'canapé'),
+('chaptéR', 'TEññiȘ');
+GO
+~~ROW COUNT: 6~~
+
+
+SELECT t1.a, t2.a FROM #test_db_collation_vu_prepare_db121_temp_t1 t1 INNER JOIN #test_db_collation_vu_prepare_db121_temp_t2 t2 on t1.a = t2.a ORDER BY t1.a;
+GO
+~~START~~
+nvarchar#!#nvarchar
+árbol#!#árbol
+café#!#café
+chaptéR#!#chaptéR
+Piñata#!#Piñata
+résumé#!#résumé
+TELÉFONO#!#TELÉFONO
+~~END~~
+
+
+-- Drop the temp tables
+DROP TABLE #test_db_collation_vu_prepare_db121_temp_t1;
+GO
+
+DROP TABLE #test_db_collation_vu_prepare_db121_temp_t2;
+GO
+
+-- Case 4.3: Table Returning Functions
+SELECT * FROM test_db_collation_vu_prepare_db121_f1();
+GO
+~~START~~
+nvarchar
+Año Nuevo
+café
+canapé
+chaptéR
+jalapeño
+naïve
+película
+résumé
+TELÉFONO
+TEññiȘ
+~~END~~
+
+
+SELECT * FROM test_db_collation_vu_prepare_db121_f2();
+GO
+~~START~~
+varchar#!#nvarchar
+TEññi?#!#TEññiȘ
+TELÉFONO#!#TELÉFONO
+résumé#!#résumé
+Piñata#!#Piñata
+película#!#película
+naïve#!#naïve
+jalapeño#!#jalapeño
+chaptéR#!#chaptéR
+canapé#!#canapé
+café#!#café
+árbol#!#árbol
+Año Nuevo#!#Año Nuevo
+~~END~~
+
+
+-- Case 4.4: SELECT INTO
+-- Create a new table named "EmployeeData" based on the "Employees" table
+SELECT * INTO test_db_collation_vu_prepare_db121_si1 FROM test_db_collation_vu_prepare_db121_t1 ORDER BY nv;
```

Comment:
YOu can write creative query which can compare column metadata of both the tables

Reply:
Added simple query to print the collation info for both the tables